src/ReactiveUI/Platforms/netstandard1.1/PlatformRegistrations.cs

```diff
@@ -1,16 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MS-PL license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-
-namespace ReactiveUI
-{
-    public class PlatformRegistrations : IWantsToRegisterStuff
-    {
-        public void Register(Action<Func<object>, Type> registerFunction)
-        {
-            throw new Exception("You are referencing the Portable version of ReactiveUI in an App. Reference the platform-specific version.");
```

Comment:
This is a big change that I'm not comfortable with. We used to throw if the end user didn't reference the right packages as part of bait and switch but now we don't. Now it'll be harder for the end user and us to troubleshoot registrations as previously we had certainty that things would explode if the user did something wrong. Now it's silent. Thoughts?